modules/administration/pages/public-cloud-azure.adoc

```diff
@@ -0,0 +1,164 @@
+[[public-cloud-azure]]
+= {productname} with Azure
+
+You can use {productname} Server and Proxy with the Microsoft Azure public cloud.
+This section discusses what you will need for running {productname} in Azure, and how to set up your installation.
+
+
+== Configure the Azure Cloud Instance
+
+Use the ``SUSE Manager Server 4 BYOS`` image.
+The image is a pre-built image created by {suse}.
+It is based on JeoS, and SUSE Manager is pre-installed but not configured.
+Configuring SUSE Manager has to be done manually with {yast}.
+
+When you create your Azure virtual machine, choose the `B2ms` size.
+The `B2ms` VM contains:
+
+* 2{nbsp}vCPUs
+* 8{nbsp}GB RAM
+* Up to 4 data disks
+* Max IOPS 2400
+* Temporary storage disk of 16{nbsp}GB.
+Data on this disk will be destroyed after the guest has been switched off.
+
```

Comment:
This isn't really consistent with the general hardware requirements listed in the Installation Guide, which lists 16 GB for base and 32 GB for production systems as well as a minimum of 4 cores.

Reply:
This is disk /dev/sdb. This is a temporary disk on Azure. I think we should not use this. Data is volatile. This is coming from Azure and is present on almost every system. For me this is a confusing disk on Azure. If you select you VMSize you will see a Temporary Storage column.

Reply:
And this is my test system. For customers I would go for at least 4 CPUs and at least 16 Gb of RAM. Sorry for not being that clear